README.md

```diff
@@ -41,13 +41,32 @@ Formats the token amount.
 #### Parameters
 
 - `options.symbol`: the token symbol, as a `String`. Overrides the value set in the constructor.
+- `options.commify`: whether the token amount should have comma separators
 - `options.digits`: the number of digits to display. Defaults to `2`.
 - `options.displaySign`: whether the sign (`-` or `+`) should be displayed for the amount.
 
 ### TokenAmount#format(options)
 
 Alias to TokenAmount#format().
 
+### TokenAmount#convert(rate, decimals, options)
+
+Converts from a rate, returning a new `TokenAmount` instance with the desired decimals and set
+options.
+
+#### Parameters
+
+- `rate`: the rate to convert from, as a `BigInt`, `String`, `Number` or `TokenAmount`.
+- `decimals`: the amount of decimals, as a `BigInt`, `String`, `Number` or `BigInt`-like (e.g. BN.js).
+- `options.symbol`: the token symbol, as a `String`. Overrides the value set in the constructor.
```

Comment:
I assume this would be the output `TokenAmount` symbol?

E.g. "convert 30 ANT at a rate of 0.3 into ETH" would produce a `TokenAmount` of 9 ETH?

Reply:
Not exactly; you'll always need to pass an options object to the `convert` function.

Reply:
I thought it would be nice to inherit from them? But maybe it’s too confusing and we should remove the support for a rate expressed as a `TokenAmount`. I was trying to write some code with it, and it feels a bit weird to attach a symbol to a rate itself, only because we want to pass it to `.convert()`. What do you guys think? :thinking:

Reply:
Update on this: coming from an offline discussion with @bpierre, we decided to remove this. Turns out it's way less elegant that we thought.